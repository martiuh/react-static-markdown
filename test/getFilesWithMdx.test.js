const getFilesWithMdx = require('../getFilesWithMdx');

test('getFilesWithMdx', () => {
  return getFilesWithMdx('test/pages/*.md').then(pages => {
    expect(pages.length).toBe(1);
    expect(pages[0]).toMatchSnapshot();
  });
});
