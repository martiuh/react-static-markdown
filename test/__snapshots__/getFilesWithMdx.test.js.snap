// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getFilesWithMdx 1`] = `
Object {
  "content": "prop value
prop Hello

<h1>Hey {value}!</h1>

# Header

Sample text

    Sample code block

<Hello value={value} />
",
  "contentmdx": "
module.exports = function({ factories={}, value, Hello }) {
  const {
    code = createFactory('code'),
    h1 = createFactory('h1'),
    p = createFactory('p'),
    wrapper = createFactory('div'),
  } = factories

  return wrapper({},

createElement(
  \\"h1\\",
  null,
  \\"Hey \\",
  value,
  \\"!\\"
),
h1({\\"id\\": \\"Header\\"},
  \\"Header\\",
),
p({},
  \\"Sample text\\",
),
code({}, \\"Sample code block\\\\n\\"),
createElement(Hello, { value: value })

  )
}
",
  "path": "/about",
  "title": "About",
}
`;
